testing/testcafe/tests/navigation/scrollable/integration.ts

```diff
@@ -2,72 +2,44 @@ import { createScreenshotsComparer } from 'devextreme-screenshot-comparer';
 import { Selector } from 'testcafe';
 import url from '../../../helpers/getPageUrl';
 import createWidget from '../../../helpers/createWidget';
-import { appendElementTo } from '../helpers/domUtils';
+import { appendElementTo, setAttribute } from '../../../helpers/domUtils';
+import { screenshotTestFn } from '../../../helpers/themeUtils';
 
-fixture`Integration_DataGrid`
+fixture.disablePageReloads`Integration_DataGrid`
   .page(url(__dirname, '../../container.html'));
 
-// T1071725
-test('The rows in the fixed column are not aligned when the grid is encapsulated inside a <td> element, useNative: false', async (t) => {
-  const { takeScreenshot, compareResults } = createScreenshotsComparer(t);
+[true, false].forEach((useNative) => {
+  test(`The rows in the fixed column are not aligned when the grid is encapsulated inside a <td> element, useNative: ${useNative} (T1071725)`, async (t) => {
+    const { takeScreenshot, compareResults } = createScreenshotsComparer(t);
 
-  await t
-    .expect(await takeScreenshot('Grid with scrollable encapsulated inside td element, useNative=false.png', Selector('#grid')))
-    .ok()
-    .expect(compareResults.isValid())
-    .ok(compareResults.errorMessages());
-}).before(async () => {
-  await appendElementTo('#container', 'table', 'outerTable', {});
-  await appendElementTo('#outerTable', 'tr', 'outerTableTR', {});
-  await appendElementTo('#outerTableTR', 'td', 'outerTableTD', {});
-  await appendElementTo('#outerTableTR', 'div', 'grid', {});
+    await screenshotTestFn(t, takeScreenshot, `Grid with scrollable wrapped in td el, useNative=${useNative}.png`, Selector('#container'));
```

Comment:
Why do we pass `Selector('#container')`, not just `'#container'`

Reply:
It doesn't matter. ScreenshotTestFn accept various value types for element: Selector | string